documentation/modules/using-must-gather.adoc

```diff
@@ -19,9 +21,7 @@ You can collect data for a one-hour or a 24-hour period and view the data with t
 .Procedure
 
 . Navigate to the directory where you want to store the `must-gather` data.
-. Run the `oc adm must-gather` command:
-
-* To gather data for the past hour:
+. Run the `oc adm must-gather` command to gather data for the past hour:
```

Comment:
"Past hour" or 24 hours is not (and was not) part of Forklift must-gather, there is no such time constraint - please remove it.

Reply:
Is forklift must gather not using the scripts in konveyor? It looks like the [default](https://github.com/konveyor/forklift-must-gather/blob/main/collection-scripts/gather_metrics) is to collect data for 6 hours.

Reply:
Correct, the metrics capture script is there, but currently it is not executed by default by the main script https://github.com/konveyor/forklift-must-gather/blob/main/collection-scripts/gather

Reply:
Yes, I see what you mean. So it looks like the default gathers data about CRs and logs but NOT metrics data. Is that correct?

Reply:
Correct!